custom/src/test/java/com/splunk/opentelemetry/micrometer/OtelNamingConventionTest.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright Splunk Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.splunk.opentelemetry.micrometer;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.BDDMockito.given;
+
+import io.micrometer.core.instrument.Meter;
+import io.micrometer.core.instrument.config.NamingConvention;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+class OtelNamingConventionTest {
+  @Mock NamingConvention namingConventionMock;
+
+  NamingConvention otelNamingConvention;
+
+  @BeforeEach
+  void setUp() {
+    otelNamingConvention = new OtelNamingConvention(namingConventionMock);
+  }
```

Comment:
Putting the class under test here I think is less readable than putting it in each of the test cases. and, it would be fewer lines of code to just put this in each of the test cases, as well.